src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/BuildAssetsUtils.cs

```diff
@@ -185,6 +185,7 @@ public static XDocument GenerateEmptyImportsFile()
 
                 new XElement(Namespace + "Project",
                     new XAttribute("ToolsVersion", "14.0"),
+                    new XAttribute("xmlns", Namespace.NamespaceName),
```

Comment:
Do you have a test in which it is explicitely tested that  the namespace is present?

Reply:
Is this necessary?
It's already set in nuget.g.props/nuget.g.targets, how does that happen?

Reply:
> Do you have a test in which it is explicitely tested that the namespace is present?

Please review [below](https://github.com/NuGet/NuGet.Client/pull/3262#discussion_r385326809) comment and let me know your feedback.